src/Pages/Login/Login.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button, Checkbox, Form, Input, Typography } from "antd";
+import { FC } from "react";
```

Comment:
can provide proper naming here {FC}

Reply:
FC stands for Functional Component. React uses this naming.